README.md

```diff
@@ -1,41 +1,45 @@
 # Open Source Project Template
 
-This repository contains a template to seed a repository for an Open Source
-project.
+# Project Title
 
-## How to use this template
+`Reactive.Annex` contains extension methods to help developers make the most out of the Reactive world.
+`Reactive.Annex.Uno` also contains an easy-to-use implementation of a dispatcher scheduler following the `IScheduler` interface.
 
-1. Check out this repository
-2. Delete the `.git` folder
-3. Git init this repository and start working on your project!
-4. Prior to submitting your request for publication, make sure to review the
-   [Open Source guidelines for publications](https://nventive.visualstudio.com/Internal/_wiki/wikis/Internal_wiki?wikiVersion=GBwikiMaster&pagePath=%2FOpen%20Source%2FPublishing&pageId=7120).
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
-## Features (to keep as-is, configure or remove)
-- [Mergify](https://mergify.io/) is configured. You can edit or remove [.mergify.yml](/.mergify.yml).
-- [allcontributors](https://allcontributors.org/) is configured. It helps adding contributors to the README.
-- [dependabot](https://dependabot.com/) is configured. This bot will open pull requests automatically to update nuget dependencies. This one could be annoying, feel free to remove the [.dependabot](/.dependabot) folder.
+## Getting Started
 
-The following is the template for the final README.md file:
+Add the `Reactive.Annex` nuget package and its dependencies to your project.
 
----
+Then, make sure you are using the `System.Reactive.Linq` and `System.Reactive.Concurrency` namespaces in your file.
 
-# Project Title
+You are now ready to use these handy extension methods!
 
-{Project tag line}
+If your project uses the [Uno platform](https://platform.uno/), you can also add the `Reactive.Annex.Uno` package to your project. You can now create a `MainDispatcherScheduler`, which implements the `IDispatcherScheduler` interface.
 
-{Small description of the purpose of the project}
+```
+var dispatcher = new MainDispatcherScheduler(CoreDispatcher.Main);
+```
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+## Features
 
-## Getting Started
+`Reactive.Annex` features:
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+* `IBackgroundScheduler` and `IDispatcherScheduler`: you no longer need to rely on argument comments to indicate whether the `IScheduler` argument should be a dispatcher or a background scheduler!
```

Comment:
argument comments?

Reply:
as an example:

https://dev.azure.com/nventive/Umbrella/_git/Umbrella.Analytics?path=%2Fsrc%2FUmbrella.Analytics.Firebase%2FFirebaseAnalyticsService.Android.cs&_a=contents&version=GBmaster

/// <summary>
		/// Initializes a new instance of the <see cref="FirebaseAnalyticsService"/> class.
		/// Android constructor for FirebaseAnalyticsService, needs current provider to log events and activity tracker for user engagements
		/// </summary>
		**/// <param name="backgroundScheduler">Background scheduler</param>
		/// <param name="dispatcherScheduler">Dispatcher scheduler</param>**
		/// <param name="contextProvider">Context provider</param>
		/// <param name="currentActivityProvider">Current activity provider</param>

How would you word it so that it's clearer what I mean?